### PR TITLE
Updating virtctl RHEL installation

### PR DESCRIPTION
**Pull request by apinnick**
Changes for 4.15: 

- Updated the virtctl binary installation title and wording so that it includes RHEL9+ and not just RHEL9.
- Updated file names.

Version(s): 4.15

Issue: None

Link to docs preview: [Installing virtctl](https://67460--docspreview.netlify.app/openshift-enterprise/latest/virt/getting_started/virt-using-the-cli-tools)

Note: The CNV repo name (cnv-4.14-for-rhel-8-x86_64-rpms) in the [Installing RPM](https://67460--docspreview.netlify.app/openshift-enterprise/latest/virt/getting_started/virt-using-the-cli-tools#virt-installing-virtctl-rhel8-rpm_virt-using-the-cli-tools) procedure uses the CNV version attribute. **It is currently 4.14 but it will be updated in 4.15 when the attribute is updated.**

QE review:
- [ ] QE has approved this change.

Additional information:

- No changes required for 4.14 (RHEL9 can be left as is). 
- This PR can be merged before 4.15 GA.

### CODE REVIEW

**Comment by dominikholler** [comment]
Looks good, thanks.

**Comment by apinnick** [comment]
@dominikholler @0xFelix  Needs SME review from one of you.

**Comment by xenolinux** [comment]
LGTM

I am new on the peer review squad. Please wait until @abhatt-rh gives my review the go-ahead

File: modules/virt-installing-virtctl-rhel8-rpm.adoc
Comment (abhatt-rh) [inline modules/virt-installing-virtctl-rhel8-rpm.adoc:9]: To be consistent with [this](https://github.com/openshift/openshift-docs/pull/67460/files#diff-c6a94af1adf62436f83e52a6e38c242ab796ff4b12e34a53b4c6c908a7ab2adeR19). 
```suggestion
You can install the `virtctl` RPM package on {op-system-base-full} 8 by enabling the {VirtProductName} repository and then installing the `kubevirt-virtctl` RPM package.
```

**Review by abhatt-rh — COMMENTED** [review]
@apinnick @xenolinux Mostly LGTM; added a minor comment

**Comment by apinnick** [comment]
@abhatt-rh  I made the change you requested. Now I need a review approval so that I can merge this PR. Thanks.

**Review by abhatt-rh — APPROVED** [review]
LGTM

**Comment by bergerhoffer** [comment]
@apinnick This was merged but not cherrypicked. Can you please cherry pick to 4.15?

**Pull request merged**
